src/devices_models/devices/common/cost_functions.jl

```diff
@@ -376,7 +376,11 @@ function ps_cost(
     gen_cost = JuMP.GenericAffExpr{Float64, _variable_type(psi_container)}()
     for (t, var) in enumerate(variable)
         if !isnothing(bin)
-            c, pwl_vars = _pwl_cost(psi_container, var, cost_array, bin[t])
+            if typeof(bin) <: ParameterJuMP.ParameterRef
```

Comment:
`if bin isa ParameterJuMP.ParameterRef` is more idiomatic.

Reply:
I'll make that change, thanks